scenarios/waymo/README.md

```diff
@@ -0,0 +1,78 @@
+# Waymo Motion Dataset
+
+SMARTS supports importing traffic histories from the Waymo motion dataset to replay in a SMARTS simulation.
+
+More information about the dataset can be found at the Waymo [website](https://waymo.com/open/data/motion/).
+
+## Setup
+
+Follow the instructions for the setup of SMARTS in the main [README](https://github.com/huawei-noah/SMARTS/). Then install the `[waymo]` dependencies to install the `waymo-dataset` package:
+
+```bash
+pip install -e .[waymo]
+```
+
+Next, download the dataset files to the folder `scenarios/waymo/waymo_dataset`. It is recommended to download the dataset files from the `training_20s` folder as they have the full traffic capture for each scenario.
```

Comment:
@RutvikGupta Perhaps, the exact steps to download dataset `training_20s` can be better explained, similar to how you had explained to me which I think was helpful.